dev-docs/modules/userId.md

```diff
@@ -1393,6 +1400,9 @@ pbjs.setConfig({
             }
         },{
             name: "pubCommonId",
+            params: {
+                pixelUrl: "/wp-json/pubcid/v1/extend/"
```

Comment:
I would drop this here in favor in including it in an example 3

Reply:
done